src/languageservice/decompressProvider.ts

```diff
@@ -4,29 +4,104 @@
  *--------------------------------------------------------------------------------------------*/
 
 import * as DecompressTar from "tar";
-import * as DecompressZip from "decompress-zip";
```

Comment:
decompress-zip is an old package that was having trouble with esbuild. I have replaced it with yauzl which is a new actively maintained un-zip lib.